tests/e2e/tests/pilot/egress_rules_test.go

```diff
@@ -50,7 +50,7 @@ func TestEgress(t *testing.T) {
 		{
 			name:              "REACHABLE_httpbin.org",
 			config:            "testdata/v1alpha1/egress-rule-httpbin.yaml",
-			url:               "http://httpbin.org/headers",
+			url:               "http://www.httpbin.org/headers",
```

Comment:
is headers a directory or file?  If it is a file, I'd recommend placing a trailing / as this removes a redirect needed.

Also, dig on www.httpbin.org results in a CNAME, but this has not propogated to google's DNS servers as of yet.  Note we have similar blocking problems in the documentation.

Reply:
headers is just a URL path, it is not a directory or file. See https://httpbin.org.